test/Node/PackageLockV3Spec.hs

```diff
@@ -0,0 +1,327 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Node.PackageLockV3Spec (
```

Comment:
[required]: Can we add tests for sub-vendored paths within workspaces?  e.g. `workspace/node_modules/a/node_modules/b`